editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -17,6 +19,18 @@ function MapAndLabelComponent(props: Props) {
         policyRef={props.policyRef}
         howMeasured={props.howMeasured}
       />
+      <MapContainer environment="standalone">
+        <my-map
+          drawMode
+          drawPointer="crosshair"
+          zoom={20}
+          drawFillColor={alpha(props.drawColour, 0.1)}
+          drawColor={props.drawColour}
+          drawPointColor={props.drawColour}
+          showCentreMarker
+          drawType={props.drawType}
+        />
```

Comment:
nit: this map is currently using a default center point in central London, but we probably want it to use the team boundary so it's a local orientation when you land here! See FindProperty's map as an example of this

I personally would also omit `showCentreMarker` as it feels a bit confusing when plotting a point!

Reply:
@jessicamcinchak so it reads in what team the flow is in and uses the ``boundaryBbox`` from the corresponding team to clip the GeoJSON?

Reply:
Yes exactly - it clips the extent of the underlying vector tiles in the basemap so they extend only a certain distance beyond the team's boundary geojson, and centers the viewport of the map based on the boundary geojson :+1:

Reply:
Cool, from the way the sidebar preview updates and when the team is set, that the team boundary will only be used when you only open the flow in a new draft, preview, or published tab? @jessicamcinchak

Reply:
Just trying to see if I should ``fetchCurrentTeam`` so the sidebar preview can have the boundary, or if it's the standard pattern to only have that specific feature in the new tab previews/drafts etc...

Reply:
@jessicamcinchak would be worth testing this new functionality, make sure I've got it working correctly with the orientation after its clipped, or if there's more I should do. I didn't end up in the middle of the local authority boundary when I tested Bucks, but this may be me having more optimistic expectations

Reply:
This works for me as expected - there's a known issue that's repeatedly come up in #help-issues related to the center being off that we need to look further into separately, but I'm generally in the council I expect and can't pan the map outside of it, which is the clip we want! https://trello.com/c/TAY09Mpw/2980-diagnose-and-fix-map-center-zoom-issue-when-displaying-team-boundaries